google/cloud/firestore_v1/__init__.py

```diff
@@ -100,6 +109,13 @@
     "__version__",
     "ArrayRemove",
     "ArrayUnion",
+    "AsyncClient",
+    "AsyncCollectionReference",
+    "AsyncDocumentReference",
+    "AsyncQuery",
+    "AsyncTransaction",
+    "async_transactional",
```

Comment:
nit: I think these may be alpha sorted?

Reply:
Good point, I thought it might make sense to group by function but alpha sorting is probably more standard.